apps/sms/views/shared/style/composer.css

```diff
@@ -186,16 +186,21 @@ form[role="search"] button[type="submit"]:after {
 #messages-send-button {
   height: 4rem;
   overflow: visible;
-  background: #fff url(images/icons/actionicon_sms_send_30x30.png)
-              no-repeat center center / 3rem;
+  background: #fff no-repeat center center / 3rem;
 
   align-self: flex-end;
-
+}
+html[dir="ltr"] #messages-send-button {
+  background-image: url(images/icons/actionicon_sms_send_30x30.png);
   /* When we put the icon at the center, an optical illusion makes it look to be
    * slightly to the left, so we offset the button by 2px to the right to
    * counteract this illusion */
   transform: translate(.2rem);
 }
+html[dir="rtl"] #messages-send-button {
+  background-image: url(images/icons/actionicon_sms_send_30x30_rtl.png);
```

Comment:
so you prefer having 2 icons instead of 1 that is mirrored ? I can understand, it makes things quite messy with the sim indicator ;)